### PR TITLE
Add AeroRust's "Space Conference - Plovdiv"

### PR DESCRIPTION
**Pull request by elpiel**
The conference will take place in Plovdiv, Bulgaria on 3rd and 4th of June. The first day will be the conference with exhibition while the second - Nanosatellite embedded workshop with Rust.

Thank you for considering our event for the newsletter!

### CODE REVIEW

File: draft/2023-05-17-this-week-in-rust.md [hunk 1]
```diff
@@ -204,6 +204,9 @@ Rusty Events between 2023-05-17 - 2023-06-14 🦀
     * [**Rust Paris meetup #59**](https://www.meetup.com/rust-paris/events/293191172)
 * 2023-05-30 | Barcelona, ES | [BcnRust](https://www.meetup.com/bcnrust/)
     * [**10th BcnRust Meetup**](https://www.meetup.com/bcnrust/events/293363107)
+* 2023-06-03 | Plovidv, BG | [AeroRust](https://aerorust.org)
+    * [**Space Conference - Plovdiv**](https://space-conference-plovdiv.eventbrite.com) (3rd of June)
+    * [**Nanosatellite embedded workshop**](https://forms.gle/rnFx5wCbrYA9qN7m6) (4th of June)
```
Comment (mariannegoldin) [inline draft/2023-05-17-this-week-in-rust.md:210]: Small formatting update:
```suggestion
* 2023-06-03 | Plovidv, BG | [AeroRust](https://aerorust.org)
    * [**Space Conference - Plovdiv**](https://space-conference-plovdiv.eventbrite.com)
* 2023-06-04 | Plovidv, BG | [AeroRust](https://aerorust.org)
    * [**Space Conference : Nanosatellite embedded workshop**](https://forms.gle/rnFx5wCbrYA9qN7m6)
```

Comment (elpiel) [reply draft/2023-05-17-this-week-in-rust.md]: LGTM

**Review by mariannegoldin — APPROVED** [review]
Thank you for this submission, we appreciate it. Approved.

**Comment by andrewpollack** [comment]
Thank you for this PR! I've gone ahead and added these events directly to the content publish as part of #4375 to ensure they make it into this weeks newsletter 🙏

**Comment by elpiel** [comment]
Thank you to both of you for including the even so quickly and in the next release of TWIR! 🥳

**Pull request closed without merging**